frontend/package.json

```diff
@@ -27,6 +27,7 @@
     "react-router-dom": "5.3.0",
     "react-scripts": "4.0.3",
     "react-sortable-hoc": "^2.0.0",
+    "styled-components": "^5.3.3",
```

Comment:
nikde nepoužíváš, pls revertni

Reply:
snažila jsme se, ale neúspšně. Teď podruhé.

Reply:
jani ne smazat, ale revertnout změnu. package.json musí zůstat, package-lock.json smaž

Reply:
prosím napiš jak: vrátím package a smažu lock - včera jsem se hecla, ale mám teplotu a úplně mi to nemyslí. Nechci to posrat

Reply:
pořešeno